### PR TITLE
Add protolint

### PR DESCRIPTION
**Pull request by fornellas**
Ditto.

---

**Stack**:
- #159
- #165
- #179 ⬅

⚠️ *Part of a stack created by [spr](https://github.com/ejoffe/spr). Do not merge manually using the UI - doing so may have unexpected results.*

### CODE REVIEW

**Comment by fornellas** [comment]
@ricardo6142dh if you know a better linter tool, I'm game, this seems functional / popular, and handles formatting.

File: internal/host/agent_server_grpc/proto/service.proto [hunk 1]
```diff
@@ -135,14 +137,14 @@ message RemoveRequest {
 message RunRequest {
   string path = 1;
   repeated string args = 2;
-  repeated string env = 3;
+  repeated string env_vars = 3;
```
Comment (fornellas) [inline internal/host/agent_server_grpc/proto/service.proto:140]: protolint wans to do `envs` (which LGTM), but it gets a bit confusing with `Env` on `Cmd`... I decided for `env_vars` instead.

File: .protolint.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+lint:
+  rules_option:
+    max_line_length:
+      max_chars: 100
```
Comment (fornellas) [inline .protolint.yaml:4]: the line:

```
option go_package = "github.com/fornellas/resonance/internal/host/agent_server_grpc/proto";
```

breaks the default=80, so I've bumped it to 100.

Comment (ricardo6142dh) [reply .protolint.yaml:4]: Ok

File: internal/host/agent_server_grpc/proto/service.proto (hunk 1 above)
Comment (ricardo6142dh) [reply internal/host/agent_server_grpc/proto/service.proto:140]: Cool

**Review by ricardo6142dh — APPROVED** [review]
Nice! LGTM

**Pull request merged**